### ISSUE TITLE
Root notebook, notebook, .... - I am confused

### ISSUE THREAD

**Issue by DutchPete** [tamlok/vnote#640]
OS Version (操作系统): Linux

VNote Version (VNote版本): Latest (最新)

Symptoms (表现): When i launched VN for the 1st time I created a notebook called *Test*. Not sure now if it was a notebook root folder or a notebook or ... ? When I launched VN yesterday it opened *Test*, today, however, it opened an empty notebook called *VNote*.
The only way I can get *Test* to open is to do it via the Explorer below the *Menu* bar. But then I cannot create a new page - the icon in the toolbar is greyed out.

I am completely confused as to how this Notebook root folder/notebook/folder/directory system works, it is not intuitive at all :confused: 
Please explain this to an enthusiastic VNote newbie.

How to Repro (如何复现):

**Comment by kerhbal** [comment]
notebook (which is a folder, everything of this notebook is in this folder, except themes, so on...)
---root folder (inside the notebook)
------everything else (inside root folder)

you can re-open your test notebook by: just input your *test* notebook. vnote will open it when the folder you input is already a notebook.
![69bbca83ly1fx0t0vmuyvg20](https://user-images.githubusercontent.com/11629684/53262502-baead780-36a4-11e9-9f2b-529fe881e104.gif)

**Comment by DutchPete** [comment]
Thank you for your reply.
Before opening this issue, I had clicked on the dropdown menu arrow next to *vnotebook*. But when I click on it there is nothing.

FYI this is how I have set up everything:
* I am using the portable version for Linux that is in a folder by itself: /home/peter/Portables/VNote/VNote-2.2-x86_64.AppImage 
* I have created a separate folder, /media/veracrypt1/Vnote/ , in which reside 3 things: my *Test* folder, a *_v_recycle_bin* folder, and *_vnote.json*.

Like I said, the only way I can get Test to open is to do it via the Explorer below the Menu bar. But then I cannot create a new page - the icon in the toolbar is greyed out.
